bootstrap.md

```diff
@@ -33,13 +33,17 @@ We leave trustless initial peer selection as future work.
 
-This is obviously impractical, so we choose to start the bootstrap sequence with nodes by connecting with an inital set of trusted peers. These can be nodes run by well-known entities (the Filecoin foundation, large exchanges, good actors in the community, etc) or manually specified by the node operator in the config file. Default Filecoin nodes will ship with a set of default nodes though these can be changed at will.
+This is obviously impractical, so we choose to start the bootstrap sequence with nodes by connecting with an inital set of trusted peers. These can be nodes run by well-known entities (the Filecoin foundation, large exchanges, good actors in the community, etc) or manually specified by the node operator in the config file. Default Filecoin nodes will ship with a set of 5 default nodes though these can be changed at will.
+
+Filecoin nodes will ship with an initial set of 5 representing "trusted" parties in the community who are deemed unlikely to try to subvert the network by providing miners with false information. It is important to note that these are meant to be 5 distinct underlying entities (as opposed to e.g. 5 nodes owned by a single party), so bootstrapping remains secure even if one or 2 is compromised. 5 is chosen as an initial number to ensure an odd number of votes in case of a disagreement, and to strike a balance between some fault-tolerance (more nodes wanted) and simplicity of bootstrapping (fewer nodes wanted).
+
+In order to secure the network, these "trusted nodes" must validate incoming blocks they receive, running as full nodes  on the network on a voluntary rather than economic basis (without participating in consensus).
 
 We leave trustless (or fully decentralized) bootstrapping for future work.
 
 A strawman bootstrapping sequence would have nodes use these initial peers to sync up to the canonical chain. However, new connection requests might effecitvely DoS these peers, making the Filecoin network unavailable to new nodes. Therefore, Filecoin bootstrapping seeks to "load-balance" initial requests to ensure the network remains widely available.
 
-We connect to each of these initial peers and then initiate the `hello handshake` (see in [network protocols](./network-protocols.md#hello-handshake)), getting back
+We connect to each of these initial peers (via DNS or IP, with a slight preference to IP connections as they are harder to coop) and then initiate the `hello handshake` (see in [network protocols](./network-protocols.md#hello-handshake)), getting back
```

Comment:
i'd specify 'hard coded IP', as you'll likely still use IP even if you use DNS to resolve it first.